pkg/test/helpers.go

```diff
@@ -175,3 +176,8 @@ func DeleteSvcs(ctx context.Context, c client.Client, vdb *vapi.VerticaDB) {
 		ExpectWithOffset(1, c.Delete(ctx, svc)).Should(Succeed())
 	}
 }
+
+// GetURTime is a helper function that returns the upgrade requeue time for the test to assert.
+func GetURTime(vdb *vapi.VerticaDB) time.Duration {
```

Comment:
I like the approach you took here, but lets take it one step further.  I think we can move this logic right into GetUpgradeRequeueTime().  So, change that function to return a `time.Duration` instead of an int.